sphinxcontrib/confluencebuilder/translator/wiki.py

```diff
@@ -562,6 +562,10 @@ def visit_term(self, node):
             self.new_state(0)
 
         self._li_terms.append(node.astext())
+        if 'ids' in node:
+            for anchor in node['ids']:
+                if self.can_anchor:
```

Comment:
(trivial; optional) You could also check for `self.can_anchor` before iterating, but I don't think it's a big performance hit.